index.html

```diff
@@ -51,22 +56,203 @@ <h1 id="principal-title" class="color-blue fonts">I’m Luis Emilio. <br> Glad t
+                                    <p class="Role">2023</p>
+                                </div>
+                                <p class="card-description">
+                                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                </p>
+                                <ul class="tags-list-style-none">
+                                    <li class="tag-description1">HTML</li>
+                                    <li class="tag-description2">CSS</li>
+                                    <li class="tag-description3">JAVASCRIPT</li>
+                                </ul>
+                                <button type="button" class="see-btn">See project</button>
+                        </div>
+                    </div>
+                </div>    
+                    <div class="grid-container">
+                        <div class="card-container">
+                            <img class="card-img" src="./vectors/Snapshoot Portfolio4.png" alt="image of my project" />
+                            <div class="text-container">
+                                <h2 class="card-title">Multi-Post <br> Stories</h3>
+                                    <div class="frame-2">
+                                        <p class="client">CANOPY</p>
+                                        <img src="./vectors/Counter.png" alt="Counter">
+                                        <p class="Role">Back End Dev</p>
+                                        <img src="./vectors/Counter.png" alt="Counter">
+                                        <p class="Role">2023</p>
+                                    </div>
+                                    <p class="card-description">
+                                        A daily selection of privately personalized reads; no accounts or sign-ups
+                                        required.
+                                    </p>
+                                    <ul class="tags-list-style-none">
+                                        <li class="tag-description1">HTML</li>
+                                        <li class="tag-description2">CSS</li>
+                                        <li class="tag-description3">JAVASCRIPT</li>
+                                    </ul>
+                                    <button type="button" class="see-btn">See project</button>
+                            </div>
+                        </div>
+    </section>
+    <div class="headline">
+        <h2 id="principal-title" class="color-blue fonts">About Myself
+        </h2>
+        <p class="p-headline fonts">I’m a software developer! I can help you build a product , feature or website Look
+            through some of my work and experience! If you like what you see and have a project you need coded, don’t
+            hesitate to contact me.</p>
+        <p class="linkheadline fonts">LET’S CONNECT</p>
+
```

Comment:
- [ ] Please wrap this sub-header inside a `<hx>` tag instead of `<p>`  because it's more semantically proper.